packages/bff/src/utilities/fileUtils.ts

```diff
@@ -97,21 +99,21 @@ export function buildJsonConfig(
       dailyCallsTotal: descriptor.dailyCallsTotal,
       description: descriptor.description,
       agreementApprovalPolicy: descriptor.agreementApprovalPolicy,
-      riskAnalysis: eservice.riskAnalysis.map((ra) => ({
-        name: ra.name,
-        riskAnalysisForm: {
-          version: ra.riskAnalysisForm.version,
-          singleAnswers: ra.riskAnalysisForm.singleAnswers.map((sa) => ({
-            key: sa.key,
-            value: sa.value,
-          })),
-          multiAnswers: ra.riskAnalysisForm.multiAnswers.map((ma) => ({
-            key: ma.key,
-            values: ma.values,
-          })),
-        },
-      })),
     },
+    riskAnalysis: eservice.riskAnalysis.map((ra) => ({
+      name: ra.name,
+      riskAnalysisForm: {
+        version: ra.riskAnalysisForm.version,
+        singleAnswers: ra.riskAnalysisForm.singleAnswers.map((sa) => ({
+          key: sa.key,
+          value: sa.value,
+        })),
+        multiAnswers: ra.riskAnalysisForm.multiAnswers.map((ma) => ({
+          key: ma.key,
+          values: ma.values,
+        })),
+      },
+    })),
```

Comment:
@Viktor-K could you check this? it seems you added risk analysis in the wrong place

Reply:
I confirm it's correct `riskAnalysis` is sibling of `descriptor` field in config JSON.[scala codebase](https://github.com/pagopa/interop-be-backend-for-frontend/blob/b99cc2f837c6ef49a2eebf757bb74ecf4bf5e9fc/src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/EServicesApiServiceImpl.scala#L947).